loomi/regulariza-morador-client
# Feature/dashboard

## Dashboard

---

**What I did:**

- Dashboard
- Finish Research Modal

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. FInish a research and see if a modal appears
3. Check if the Dashboard looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
4. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/25696006/121825492-2481bc00-cc89-11eb-8e04-dfb3c298380b.png)
![image](https://user-images.githubusercontent.com/25696006/121825496-29df0680-cc89-11eb-96e3-a701cee93b18.png)
